docs/components/tabs.md

```diff
@@ -0,0 +1,348 @@
+**`MaterialToolbar` icon color** |  In `TabBar` assign the `labelColor` parameter
+
+### Active text label attributes
+
+&nbsp;         | Property
+-------------- | ------------------------
+**Text label** | In widget `Tab` assign the `text` parameter.
+**Color**      | In `TabBar` assign the `labelColors` and `unselectedLabelColor` parameter
+**Typography** | In `TabBar` assign the `labelStyle` and `unselectedLabelStyle` parameter
+
+### Active tab indicator attributes
+
+&nbsp;         | Property
+-------------- | ------------------------
+**Indicatorg** | Indicat exist be default for custom indicator go to `TabBar` widget and fill out the `indicator` attribute.
+**Indicator Padding** | In widget `TabBar` assign the `indicatorPadding` parameter.
+**Indicator Size**      | In `TabBar` assign the `indicatorSize` parameter
+**Indicator Weight** | In `TabBar` assign the `indicatorWeight` parameter
+
+### Inactive  icon attributes
+
+&nbsp; | Property
+------ | ---------
+**`MaterialToolbar` icon** | In widget `Tab` assign the `icon` parameter
+**`MaterialToolbar` icon color** |  In `TabBar` assign the `unselectedLabelColor` parameter
+
+
+### Inactive text label attributes
+
+&nbsp;         | Property
+-------------- | ------------------------
+**Text label** | In widget `Tab` assign the `text` parameter.
+**Color**      | In `TabBar` assign the `unselectedLabelColor` parameter
+**Typography** | In `TabBar` assign the ``unselectedLabelStyle` parameter
+
+### Tab item attributes
+
+&nbsp;         | Property
+-------------- | ------------------------
+**child** | In widget `Tab` assign the `child` parameter. This is usually a `Text` widget, possibly wrapped in a `Semantics` widget for custom accessibility.
+
+## Fixed tabs
+
+Fixed tabs display all tabs on one screen, with each tab at a fixed width. The width of each tab is determined by dividing the number of tabs by the screen width. They don’t scroll to reveal more tabs; the visible tab set represents the only tabs available.
+
+### Fixed tab example
+
+`Tabs`
+* [Class definition](https://api.flutter.dev/flutter/material/TabBar-class.html)
+* [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/tabs.dart)
+* [Dartpad demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/tabs/fixed)
+* [Demo site](https://gallery.flutter.dev/#/demo/tabs)
+
+The following example shows a row of of fixed tabs.
+
+![Fixed tab example with 3 tabs, each tab has an icon](assets/tabs/tabs_fixed.png)
+
+```dart
+MaterialApp(
+      title: 'Flutter Demo',
+      home: DefaultTabController(
+        length: 3,
+        child: Scaffold(
+          appBar: AppBar(
+            title: Text('Fixed Tabs'),
+            automaticallyImplyLeading: false,
+            backgroundColor: Color(0xff5808e5),
+            bottom: TabBar(
```

Comment:
Add the indicator color to all of the examples in this file, so that it looks better. As mentioned in the dart pad demo code

Reply:
Done